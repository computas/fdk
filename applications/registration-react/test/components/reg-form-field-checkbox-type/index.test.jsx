import React from 'react';
import { shallow } from 'enzyme';
import CheckboxFieldType from '../../../src/components/reg-form-field-checkbox-type';

let defaultProps;
let wrapper;

beforeEach(() => {
  defaultProps = {
    input: { name: 'type', value: [] }
  };
  wrapper = shallow(<CheckboxFieldType {...defaultProps} />);
});

test('should render CheckboxFieldType correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
