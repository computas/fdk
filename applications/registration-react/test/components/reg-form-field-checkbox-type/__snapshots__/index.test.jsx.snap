// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CheckboxFieldType correctly 1`] = `
<div>
  <label
    className="form-check fdk-form-checkbox"
    htmlFor="Data"
    key="0"
  >
    <input
      checked=""
      id="Data"
      name="types"
      onChange={[Function]}
      type="checkbox"
      value="Data"
    />
    <span
      className="form-check-label fdk-form-check-label"
    />
    <span>
      Data
    </span>
  </label>
  <label
    className="form-check fdk-form-checkbox"
    htmlFor="Kodelister"
    key="1"
  >
    <input
      checked=""
      id="Kodelister"
      name="types"
      onChange={[Function]}
      type="checkbox"
      value="Kodelister"
    />
    <span
      className="form-check-label fdk-form-check-label"
    />
    <span>
      Kodelister
    </span>
  </label>
  <label
    className="form-check fdk-form-checkbox"
    htmlFor="Tesauri"
    key="2"
  >
    <input
      checked=""
      id="Tesauri"
      name="types"
      onChange={[Function]}
      type="checkbox"
      value="Tesauri"
    />
    <span
      className="form-check-label fdk-form-check-label"
    />
    <span>
      Tesauri
    </span>
  </label>
  <label
    className="form-check fdk-form-checkbox"
    htmlFor="Taksonomi"
    key="3"
  >
    <input
      checked=""
      id="Taksonomi"
      name="types"
      onChange={[Function]}
      type="checkbox"
      value="Taksonomi"
    />
    <span
      className="form-check-label fdk-form-check-label"
    />
    <span>
      Taksonomi
    </span>
  </label>
  <label
    className="form-check fdk-form-checkbox"
    htmlFor="Testdata"
    key="4"
  >
    <input
      checked=""
      id="Testdata"
      name="types"
      onChange={[Function]}
      type="checkbox"
      value="Testdata"
    />
    <span
      className="form-check-label fdk-form-check-label"
    />
    <span>
      Testdata
    </span>
  </label>
</div>
`;
